xla/hlo/transforms/host_offloader.cc

```diff
@@ -263,7 +263,9 @@ absl::StatusOr<bool> HostOffloader::WalkDownHostMemoryOffloadPaths(
       LOG(WARNING) << absl::StreamFormat(
           "Found an instruction (\"%s\") which does device compute in host "
           "memory space. Converting into host compute. This is likely to have "
-          "a very high overhead.",
+          "a very slow execution time. If you're using JAX, use device_put() "
+          "on the inputs to move the computation to the "
```

Comment:
Can we make the wording slightly more accurate? `device_put` does not "move computation" anywhere. A more accurate alternative:
"use device_put() to move the inputs to the device so that computation happens on the device."